pkg/chartvalues/aws_operator.go

```diff
@@ -93,8 +100,8 @@ func NewAWSOperator(config AWSOperatorConfig) (string, error) {
 	if config.Secret.AWSOperator.SecretYaml.Service.AWS.HostAccessKey.Secret == "" {
 		return "", microerror.Maskf(invalidConfigError, "%T.Secret.AWSOperator.SecretYaml.Service.AWS.HostAccessKey.Secret must not be empty", config)
 	}
-	if config.RegistryPullSecret == "" {
-		return "", microerror.Maskf(invalidConfigError, "%T.RegistryPullSecret must not be empty", config)
+	if config.SSH.UserList == "" {
+		return "", microerror.Maskf(invalidConfigError, "%T.SSH.UserList must not be empty", config)
 	}
```

Comment:
The SSH UserList is now injected so that we do not hard code it in the `aws-operator` anymore.